src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -1277,15 +1277,29 @@ const deviceParameterConvertersMap = {
 };
 
 const getRequestChannelParameter = ( buffer: ICommandBinaryBuffer ): IRequestChannelParameter => (
-    {
-        channel: getChannelValue(buffer)
-    }
+    {channel: getChannelValue(buffer)}
 );
 
 const setRequestChannelParameter = ( buffer: ICommandBinaryBuffer, parameter: IRequestChannelParameter ) => {
     setChannelValue(buffer, parameter.channel);
 };
 
+const getRequestDataTypeParameter = ( buffer: ICommandBinaryBuffer ): IRequestDataTypeParameter => (
```

Comment:
do we need these 4 functions? they are used once each

Reply:
functions removed